specification/confidentialledger/ConfientialLedger/cadl-project.yaml

```diff
@@ -0,0 +1,10 @@
+emitters:
+  "@azure-tools/cadl-python":
+    "sdk-folder": sdk/confidentialledger/azure-confidentialledger
+  "@azure-tools/cadl-csharp":
+    "sdk-folder": sdk/confidentialledger/Azure.Security.ConfidentialLedger
+  "@azure-tools/cadl-typescript":
+    "sdk-folder": sdk/confidentialledger/azure-confidentialledger-rest
+  "@azure-tools/cadl-java":
+    "sdk-folder": sdk/confidentialledger/azure-confidentialledger
```

Comment:
java need one additional configure here

```
  "@azure-tools/cadl-java":
    "sdk-folder": sdk/confidentialledger/azure-confidentialledger
    namespace: com.azure.confidentialledger
```

the `namespace` here need to match the `sdk-folder`.

officially the package is `azure-security-confidentialledger`, but don't use that in test as there is already such package in Java SDK repo.

Reply:
updated. If you need any other changes, please change this PR directly. thx